### DISCUSSION TITLE
Chakra ui styles in components not working when using resetCss={false}

### DISCUSSION THREAD

**Discussion by fedelombar** [chakra-ui/chakra-ui#5751] [General]
When I try to use styles of manners, box, chakra etc. It brings me everything without styles. This has happened when in the provider of ChakraProvider I have used the resetCss so that my css of the app does not change once chakra is installed.
For example when I want to use a Chakra component, it recognizes the component but gives it to me without styles.
I haven't found a solution yet for this problem. Has anyone had this happen to you? How have you solved it?

My code provider:

```
function App () {
  AOS.init()
  return (

    <Provider store={store}>
      <ChakraProvider resetCSS={false}>
      <AppRouter />
      <FloatingZendesk/>
      </ChakraProvider>
    </Provider>
  )
}
```

**Comment by sahmmie** [comment]
@fedelombar Were you able to solve this?
